extension/src/popup/helpers/soroban.ts

```diff
@@ -186,3 +201,33 @@ export const getAttrsFromSorobanOp = (
     ...opArgs,
   };
 };
+
+export const getAttrsFromSorobanTxOp = (operation: HorizonOperation) => {
```

Comment:
we now have 2 helper methods to decode a soroban tx as the tx built from an xdr is _slightly_ different than what Horizon provides. Maybe this can be combined once I understand this a little better

Reply:
Breaking out the shared rootInvocation stuff into its own function